### ISSUE TITLE
error detected while processing ~/.vim/bundle/vim-gitgutter/plugin/gitgutter.vim:

### ISSUE THREAD

**Issue by rainysia** [airblade/vim-gitgutter#773]
Hi there,

vim-gitgutter: f2a825f
The others version which is older than f2a825f works fine.

OS: debian 10.8
VIM - Vi IMproved 8.1 (2018 May 18, compiled Jun 15 2019 16:41:15)
Included patches: 1-875, 878, 884, 948, 1046, 1365-1368, 1382, 1401

After upgrade the vim-gitgutter, it pops the error when vim initial
```
Error detected while processing ~.vim/bundle/vim-gitgutter/plugin/gitgutter.vim:
line   28:
E113: Unknown option: previewpopup
E116: Invalid arguments for function empty(&previewpopup))
E116: Invalid arguments for function get(g:, 'gitgutter_preview_win_floating', !empty(&previewpopup))
```
Any suggestions? Thx.

**Comment by its-izhar** [comment]
Got the exact the same error after upgrading today.

**Comment by airblade** [comment]
Fixed in 7224d1d.
